enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/IdentityVerificationService.java

```diff
@@ -304,6 +298,11 @@ public void checkVerificationResult(IdentityVerificationPhase phase, OwnerId own
             return;
         }
 
+        if (!requiredDocumentTypesGuard.evaluate(idVerification.getDocumentVerifications(), idVerification.getId())) {
```

Comment:
Here there is a guard to stop until all required documents are present and accepted.